src/MagnumPlugins/BasisImporter/BasisImporter.cpp

```diff
@@ -239,9 +239,7 @@ Containers::Optional<ImageData2D> BasisImporter::doImage2D(UnsignedInt index) {
        blows up for someone, we can reconsider. */
     CORRADE_INTERNAL_ASSERT_OUTPUT(_state->transcoder->get_image_level_desc(_state->in.data(), _state->in.size(), index, level, origWidth, origHeight, totalBlocks));
 
-    /* cDecodeFlagsPVRTCWrapAddressing is the default used by
-       transcode_image_level() */
-    const std::uint32_t flags = basist::basisu_transcoder::cDecodeFlagsPVRTCWrapAddressing;
+    const std::uint32_t flags = 0;
```

Comment:
Yeah, Rich removed this two hours after I added that here :D I'll commit this soon into `master` (and bump the revisions for CIs).

Reply:
Done: 66f73571db6b47c0e322cca0717f56cdce2de3df